webapp-django/crashstats/authentication/management/commands/auditgroups.py

```diff
@@ -18,6 +20,19 @@
 VALID_EMAIL_DOMAINS = ('mozilla.com', 'mozilla.org')
 
 
+def get_or_create_auditgroups_user():
+    try:
+        return User.objects.get(username='auditgroups')
+    except User.DoesNotExist:
+        return User.objects.create_user(
+            username='auditgroups',
+            email='auditgroups@example.com',
+            first_name='SYSTEMUSER',
+            last_name='DONOTDELETE',
+            is_active=False,
+        )
```

Comment:
So what's going on here is that in order to add an entry to `LogEntry`, it has to be from a user. So the auditgroups command needs to have a user to add things with. This creates a "system user" that's inactive to use for creating `LogEntry` items. That way when auditgroups removes someone from the group, we end up with an audit trail.